guides/common/modules/proc_enabling-remote-execution.adoc

```diff
@@ -0,0 +1,24 @@
+[id="enabling-remote-execution_{context}"]
+= Enabling Remote Execution
+
+Use this procedure to enable remote execution on your {SmartProxyServer}.
+To learn more about remote execution, see {ManagingHostsDocURL}Configuring_and_Setting_Up_Remote_Jobs_managing-hosts[Configuring and Setting Up Remote Jobs] in _{ManagingHostsDocTitle}_.
+
+ifdef::foreman-el,foreman-deb[]
+.Prerequisite
+* You have enabled the remote execution plug-in on your {ProjectServer}.
```

Comment:
Would it make sense to introduce a similar part in the Installing Server guide? Perhaps even link there instead of inlining?

Reply:
I suppose it would be good to add this section; however; the entire chapter dealing with remote execution is currently in [Managing Hosts](https://docs.theforeman.org/nightly/Managing_Hosts/index-foreman-el.html#Configuring_and_Setting_Up_Remote_Jobs_managing-hosts), so I think it would be the best to add it there. I am thinking to add it after the **Remote Execution Workflow** chapter. What do you think?

Reply:
A good question, and perhaps it comes down to preference. I think there is value in having it in the install guides (both server and proxy). Even if they're small, I really like cross references. But if you it for one plugin, should we do it for all? For example, you could make the same argument for OpenSCAP.

Perhaps @Lennonka or @asteflova can weigh in here since it's less about technical details (my specialty) and more about a bigger content strategy.

Reply:
I see this could turn into a work of potentially large scope. I will merger this PR and open an issue where we can discuss the next steps.